devp2p/peermanager.py

```diff
@@ -210,8 +211,7 @@ def stop(self):
         super(PeerManager, self).stop()
 
 
-class PeerErrors(object):
-
+class PeerErrorsBase(object):
     def __init__(self):
```

Comment:
`__init__` especially with exit should be implemented in `PeerErrors`, not in the Base. Also `report`.